api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityStatusCalculator.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.dhsc.htbhf.claimant.service;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.dhsc.htbhf.claimant.model.eligibility.EligibilityResponse;
+import uk.gov.dhsc.htbhf.claimant.repository.ClaimantRepository;
+import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
+
+@Service
+@AllArgsConstructor
+public class EligibilityStatusCalculator {
```

Comment:
Now this is out into its own class, I think some JavaDoc describing the rules it applies would be useful.

Reply:
Added javadoc.